src/live_restore/live_restore_fs.c

```diff
@@ -840,22 +835,33 @@ __live_restore_fill_hole(WT_FILE_HANDLE *fh, WT_SESSION *wt_session, bool *finis
         return (0);
     }
 
-    __wt_verbose_debug3(session, WT_VERB_FILEOPS,
+    __wt_verbose_debug3(session, WT_VERB_LIVE_RESTORE,
       "Found hole in %s at %" PRId64 "-%" PRId64 " during background migration. ", fh->name,
       hole->off, WT_EXTENT_END(hole));
 
     /*
      * When encountering a large hole, break the read into small chunks. Split the hole into n
      * chunks: the first n - 1 chunks will read a full WT_LIVE_RESTORE_READ_SIZE buffer, and the
-     * last chunk reads the remaining data. This loop is a not obvious, effectively the read is
+     * last chunk reads the remaining data. This loop is not obvious, effectively the read is
      * shrinking the hole in the stack below us. This is why we always read from the start at the
      * beginning of the loop.
      */
     char buf[WT_LIVE_RESTORE_READ_SIZE];
     size_t read_size = WT_MIN(hole->len, (size_t)WT_LIVE_RESTORE_READ_SIZE);
-    __wt_verbose_debug2(session, WT_VERB_FILEOPS,
+    uint64_t time_diff_ms;
+
+    __wt_verbose_debug2(session, WT_VERB_LIVE_RESTORE,
       "    BACKGROUND READ %s : %" PRId64 ", %" WT_SIZET_FMT, lr_fh->iface.name, hole->off,
       read_size);
+    __wt_timer_evaluate_ms(session, start_timer, &time_diff_ms);
+    if ((time_diff_ms / (WT_THOUSAND * WT_PROGRESS_MSG_PERIOD)) > *msg_count) {
+        __wt_verbose(session, WT_VERB_LIVE_RESTORE_PROGRESS,
+          "Live restore running on %s for %" PRIu64
```

Comment:
This message shares the same timer as `Live restore has been running for XXX milliseconds and has processed file Y of Z` so we'll print either that message or this one every 20 seconds. Is it intentional that we randomly switch between the two messages?

Reply:
AFAICT, `live_restore_fill_hole` is only called from `wti_live_restore_fs_fill_holes` in the `!finished` loop. That uses a local variable to keep track of the 20 seconds for processing that one file. So it is independent of the other 20 second messaging (and will be independent of other worker threads processing their files too). Am I missing something?

Reply:
I think this is okay given this is file specific and the other message is general.

Reply:
Ah yep that's my mistake. I misread the code and thought this was also using `server->start_timer`.